core/Cargo.toml

```diff
@@ -279,7 +285,7 @@ sha1 = { version = "0.10.6", optional = true }
 sha2 = { version = "0.10", optional = true }
 
 # For http based services.
-reqsign = { version = "0.14.7", default-features = false, optional = true }
```

Comment:
Pin it to a commit should be better?

Reply:
We will continue after reqsign making a new release.